packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -161,6 +161,18 @@ tbody.spectrum-Table-body {
 
   &.spectrum-Table-row--highlightSelection {
     border-bottom-color: var(--spectrum-global-color-blue-500);
+    &.is-selected {
+      border-inline-start: 1px solid var(--spectrum-global-color-blue-500);
+      border-inline-end: 1px solid var(--spectrum-global-color-blue-500);
+      &.spectrum-Table-row--firstRow {
+        border-block-start: 1px solid var(--spectrum-global-color-blue-500);
+      }
```

Comment:
Causing a wiggle cuz the border is only there when the highlight selection TableView row is selected. Tried reserving the room by adding a transparent border at all times but then this makes the focus indicator misplaced...